packages/input-manager/src/inputManager.ts

```diff
@@ -33,11 +33,12 @@ type SomeDeviceConfig =
 	| DeviceConfig<DeviceType.SKAARHOJ, SkaarhojDeviceConfig>
 	| DeviceConfig<DeviceType.OSC, OSCDeviceConfig>
 
-interface TriggerEventArgs extends DeviceTriggerEventArgs {
+export interface ManagerTriggerEventArgs {
 	/** The ID of the device that issued this event */
 	deviceId: string
-	/** Should this event replace whatever unsent events there are */
-	replacesPrevious?: boolean
+
+	/** Callback to retrieve the next trigger, to be used when it's time to send the trigger to Core. */
```

Comment:
I don't think this is technically a callback? It's a method to get the latest trigger.